test/fixtures/msdos-btrfs-shadowed-twice.ycp

```diff
@@ -0,0 +1,246 @@
+$[
```

Comment:
NP: Could we get rid of the YCP data format? I know it's a bit out of scope of this PR but we should finally kill the last `*.ycp` zombies...

BTW there is a [`ycp2yml.rb` conversion tool](https://github.com/yast/yast-devtools/blob/master/ytools/yast2/ycp2yml.rb)...